recipe/environment_isofit_arm64.yml

```diff
@@ -0,0 +1,35 @@
+name: isofit_env
+
+channels: &id1
+  - https://anaconda.org/apple # Mac M1s
```

Comment:
I guess this is a typo. The channel's address needs to be `http://conda.anaconda.org/apple` in order to work properly.